tests/lib/external/snapd-testing-tools/tools/tests.pkgs.apt.sh

```diff
@@ -79,3 +79,7 @@ cmd_remove() {
     # shellcheck disable=SC2086
     apt-get remove --yes $REMOVE_FLAGS "$@"
 }
+
+cmd_download() {
+    apt download $@ >/dev/null 2>&1
```

Comment:
It is generally incorrect to use $@ without "$@".

```suggestion
    apt download "$@" >/dev/null 2>&1
```

Reply:
I had to add "# shellcheck disable=SC2068" to allow downloading several packages with the same command
Also updated the test to check that